### PR TITLE
Harolddavis/ch107770/fix up hierarchy folder structure in the

### PR DESCRIPTION
**Pull request by HaroldFS**
Fix up Hierarchy so everything is nice an organized …

### CODE REVIEW

**Comment by patrick-fs** [comment]
Hey @HaroldFS , I think you may have fixed up the hierarchy already on your latest PR: https://github.com/fullstorydev/ios-shoppe-demo/pull/5

Does the `master` branch looks right to you?

**Comment by HaroldFS** [comment]
> Hey @HaroldFS , I think you may have fixed up the hierarchy already on your latest PR: #5
> 
> Does the `master` branch looks right to you?

Actually I meant to say "improve"  :), master is not the way I want it. I want the files to be organized into major folders

File: swift/ios-shoppe-demo/Resources/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//
+//  AppDelegate.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/3/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
```
Comment (patrick-fs) [inline swift/ios-shoppe-demo/Resources/AppDelegate.swift]: I'm not used to seeing code in a "resources" directory. From [this](https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFBundles/BundleTypes/BundleTypes.html#//apple_ref/doc/uid/10000123i-CH101-SW1):

> Resources are data files that live outside your application’s executable file. Resources typically consist of things like images, icons, sounds, nib files, strings files, configuration files, and data files (among others)

Should `AppDelegate.swift` and `SceneDelegate.swift` live here?

**Comment by patrick-fs** [comment]
> Actually I meant to say "improve" :), master is not the way I want it. I want the files to be organized into major folders

Got it. I got confused because it looks like some commits that are already on master are in this branch as well.

image

Would you mind rebasing from master onto this branch?

**Comment by HaroldFS** [comment]
> > Actually I meant to say "improve" :), master is not the way I want it. I want the files to be organized into major folders
> 
> Got it. I got confused because it looks like some commits that are already on master are in this branch as well.
> 
> image
> 
> Would you mind rebasing from master onto this branch?

Done :)

File: swift/ios-shoppe-demo/Resources/AppDelegate.swift (hunk 1 above)
Comment (HaroldFS) [reply swift/ios-shoppe-demo/Resources/AppDelegate.swift]: Maybe not. I moved them out.

**Comment by patrick-fs** [comment]
@HaroldFS It looks like some more cruft may have been committed:
image

**Comment by HaroldFS** [comment]
> @HaroldFS It looks like some more cruft may have been committed:
> image

That was weird but it should be resolved now.

File: swift/ios-shoppe-demo/Resources/AppDelegate.swift (hunk 1 above)
Comment (patrick-fs) [reply swift/ios-shoppe-demo/Resources/AppDelegate.swift]: It looks like `AppDelegate.swift` and `SceneDelegate.swift` (just noticed `Utilities.swift` as well) are still in the `Resources` directory:

image
image

Maybe you forgot to commit the change?

**Comment by HaroldFS** [comment]
> @HaroldFS It looks like some more cruft may have been committed:
> image

Looks like when moving items around they where copied instead of moved. I removed the duplicates and ran the project to make sure I'm not missing anything. Done.

File: swift/ios-shoppe-demo/ViewController.swift [hunk 2]
```diff
@@ -8,13 +8,15 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class StoreViewController: UICollectionViewController {
```
Comment (patrick-fs) [inline swift/ios-shoppe-demo/ViewController.swift:11]: Hey @HaroldFS it looks like you've got and identical class in 2 files: `ViewController.swift` and `StoreViewController.swift`. Was this intentional?

File: swift/ios-shoppe-demo/Resources/AppDelegate.swift (hunk 1 above)
Comment (patrick-fs) [reply swift/ios-shoppe-demo/Resources/AppDelegate.swift]: `Utilities.swift` is still in the `Resources` directory:

image

 
image

Comment (HaroldFS) [reply swift/ios-shoppe-demo/Resources/AppDelegate.swift]: It should be there its needed.

File: swift/ios-shoppe-demo/ServiceManagers/APIService.swift [hunk 3]
```diff
@@ -0,0 +1,46 @@
+//
+//  APIService.swift
```
Comment (patrick-fs) [inline swift/ios-shoppe-demo/ServiceManagers/APIService.swift]: @HaroldFS this looks like another duplicated file (also at `swift/ios-shoppe-demo/APIService.swift`). Would you please verify:

image

**Review by patrick-fs — APPROVED** [review]
LGTM

**Pull request merged**